tools/firmware-utils/src/jcgimage.c

```diff
@@ -316,6 +323,8 @@ main(int argc, char **argv)
 	char *file1 = NULL;
 	char *file2 = NULL;
 	char *version = NULL;
+	size_t maxsize = MAXSIZE;
+	char* endptr;
```

Comment:
please follow the style of "char *file..." and make change this to

    char *endptr;

(scripts/checkpatch.pl catches stuff like this)